sdk/typescript/introspector/scanner/typescript_module/explorer.ts

```diff
@@ -0,0 +1,23 @@
+import Module from "node:module"
+
+export function findModuleByExportedName(
```

Comment:
What if two modules have the same exported name?

Reply:
No, I was thinking you can rename on import but Python doesn't handle that as well. It uses the name of the class itself, not the name it's imported with.

Reply:
I think the TS SDK doesn't handle it as well.

Reply:
This isn't resolved because there's an unaswered question:

> What kinds of things goes through this function?

Will not block though, and let users raise any issues and deal with them when they arrive.